test/specs/point.spec.js

```diff
@@ -1,92 +1,55 @@
 describe('Point annotation', function() {
   describe('auto', jasmine.fixtures('point'));
 
-  const eventIn = function(xScale, yScale, element) {
-    const options = element.options;
-    const adjust = options.borderWidth / 2 - 1;
-    return {x: element.x, y: element.y - element.height / 2 - adjust};
-  };
-  const eventOut = function(xScale, yScale, element) {
-    const options = element.options;
-    const adjust = options.borderWidth / 2 + 1;
-    return {x: element.x, y: element.y - element.height / 2 - adjust};
-  };
-
-  window.testEvents({
-    type: 'point',
-    id: 'test',
-    xValue: 5,
-    yValue: 5,
-    radius: 30,
-    borderWidth: 12
-  }, eventIn, eventOut);
-
-  describe('events, removing borderWidth by callback, ', function() {
-    const chartConfig = {
-      type: 'scatter',
-      options: {
-        animation: false,
-        scales: {
-          x: {
-            display: false,
-            min: 0,
-            max: 10
-          },
-          y: {
-            display: false,
-            min: 0,
-            max: 10
-          }
-        },
-        plugins: {
-          legend: false,
-          annotation: {
-            annotations: {
-              point: {
-                type: 'point',
-                xValue: 5,
-                yValue: 5,
-                radius: 20,
-                borderWidth: 10
-              }
-            }
-          }
-        }
-      },
+  describe('inRange', function() {
+    const annotation = {
+      type: 'point',
+      xValue: 5,
+      yValue: 5,
+      radius: 30
```

Comment:
Could test couple of radiuses too. For test performance, its better to have couple of points in the same chart with different radiuses and test each of those.

Reply:
I agree. Do you want I will add the tests on 2 points in this PR?

Reply:
Done!